### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#48]
As a buyer or seller, I need to orient myself with the OASIS+ solution, so that I can determine appropriate next steps. 

- [Content](https://docs.google.com/document/d/1GiZ1xUr4rGSoCXn4VxYYNMcuJZ0vfxuusaqXKHRaCnM/edit#heading=h.iygv7gu3pcbe): 
- [Design](https://www.figma.com/proto/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?page-id=1448%3A4471&type=design&node-id=2389-11651&viewport=62%2C442%2C0.03&t=wCuBrSBFXdhy5RH7-1&scaling=min-zoom&starting-point-node-id=2111%3A11279)

**Status**

- [x] Content created
- [x] Content reviewed by SD
- [x] Content reviewed by SME
- [x] Design finalized
- [x] Content finalized
- [x] Page developed

**Comment by klohman** [comment]
@ekidenda Mark and I brainstormed and our other alerts use bulleted lists, and I think this solves the spacing problem. Does this work for you?
<img width="563" alt="Screenshot 2024-02-22 at 4 14 00 PM" src="https://github.com/GSA/oasis-plus/assets/18366192/494395bb-691f-4ae7-b0d1-292589f71032">

**Comment by ekidenda** [comment]
@klohman Yes, it works! Thank you for checking on it and finding a solution.

**Comment by PowderD** [comment]
[image: image.png]

*Jon DeWolfe  |  *
Program Manager Buyer Engagement
Services Marketplace
Federal Acquisition Service, U.S. General Services Administration
|(: 208.830.8517 | *: ***@***.***

**Comment by vickimcfadden** [comment]
Hi @PowderD could we wait on implementing the centered text on `find out what's next` until post-MVP?

**Comment by vickimcfadden** [comment]
Considering this done!
